low-level/framework/utils/service_logging.py

```diff
@@ -19,130 +19,23 @@
  ****************************************************************************
 """
 
-import logging.handlers
-import time
+import sys
 import os
+from cortx.utils.log import Log
 
+def init_logging(dcs_service_name, file_path, log_level="INFO"):
+    """Initialize logging for SSPL component."""
+    try:
+        Log.init(service_name=dcs_service_name, log_path=file_path, level=log_level)
+    except Exception as err:
+        print("[ Error ] when initializing logging :")
+        print(err)
+        print("Exiting ...")
+        sys.exit(os.EX_USAGE)
```

Comment:
Also why exiting with EX_USAGE, when it may not be usage related error ! EX_SOFTWARE  seems proper

>	EX_USAGE -- The command was used incorrectly, e.g., with
 \*		the wrong number of arguments, a bad flag, a bad
 \*		syntax in a parameter, or whatever.

> 	EX_SOFTWARE -- An internal software error has been detected.
 \*		This should be limited to non-operating system related
 \*		errors as possible.

Reply:
same comment. This was the old code,  just moved it to service_logging file from sspl_ll_d.
But Agree, EX_SOFTWARE makes more sense,
optimized the code